build.gradle

```diff
@@ -25,7 +25,15 @@ java {
 }
 
 def versions = [
+        junit           : '5.7.2',
+        junitPlatform   : '1.7.2',
+        jsr310          : '2.13.2',
+        lombok          : '1.18.24',
+        reformLogging   : '5.1.7',
         springBoot      : springBoot.class.package.implementationVersion,
+        springCloud     : '2021.0.2',
+        springfoxSwagger: '3.0.0',
+        serviceAuthVersion: '4.0.3',
```

Comment:
I think we can delete all these apart from `lombok` that are now showing as new changes under this PR? (Presumably added back in as part of a merge conflict!)

Reply:
Yes, merge conflict - fixed in https://github.com/hmcts/ts-translation-service/pull/5/commits/6acfcae56a6cab21d7228b8bccc44c3b51ff2a75